test/schematype.test.js

```diff
@@ -208,6 +208,25 @@ describe('schematype', function() {
     });
   });
 
+  it('merges default validators (gh-14070)', function() {
+    class TestSchemaType extends mongoose.SchemaType {}
+    TestSchemaType.set('validate', v => typeof v === 'string');
```

Comment:
maybe consider making it a stand-alone function and later check against it being still contained